examples/sshguard.conf.sample

```diff
@@ -9,12 +9,17 @@
 #BACKEND="/usr/local/libexec/sshg-fw-iptables"
 
 # Space-separated list of log files to monitor. (optional, no default)
-#FILES="/var/log/auth.log /var/log/authlog /var/log/maillog"
+# debian-style systems
+#FILES="/var/log/mail.log /var/log/auth.log"
+# slackware and redhat-style systems
+#FILES="/var/log/maillog /var/log/secure"
 
 # Shell command that provides logs on standard output. (optional, no default)
-# Example 1: ssh and sendmail from systemd journal:
+# ssh and sendmail from systemd journal
```

Comment:
Let's keep enumerating the examples. That way, it's clear that you can only have one set at a time.